js/components/addressBookList/addressBookList_controller.js

```diff
@@ -7,6 +7,14 @@ angular.module('contactsApp')
 	AddressBookService.getAll().then(function(addressBooks) {
 		ctrl.addressBooks = addressBooks;
 		ctrl.loading = false;
+		if(ctrl.addressBooks.length === 0) {
+			AddressBookService.create(t('contacts', 'Contacts')).then(function() {
```

Comment:
Dayum, the promise returned by the service does not pass the new addressbook when it's resolved. Therefore we cannot simply use the above code snippet. Well, let's go with your approach then :-)